frontend/common/src/components/form/TextArea/TextArea.tsx

```diff
@@ -24,15 +26,25 @@ const TextArea: React.FunctionComponent<TextAreaProps> = ({
   name,
   rules = {},
   children,
+  whitespaceTrim = true,
   ...rest
 }) => {
   const {
     register,
     formState: { errors },
+    setValue,
   } = useFormContext();
   // To grab errors in nested objects we need to use lodash's get helper.
   const error = get(errors, name)?.message;
 
+  const whitespaceTrimmer = () => {
```

Comment:
Since we reuse this, it could be useful to extract this into a utility/helper function.

Reply:
in order to use `trim()` I have to constrain things to `type: string` , so I am not experienced enough to know how to write this out so genericly
trim is a string method for instance so I don't think I can write things out to return an arbitrary `type V`
😖

Reply:
This was just a suggestion, if we use the `React.FocusEvent<HTMLInputElement>` mentioned in the other comment, it think this function could be simple enough to not warrant extracting it.

Reply:
got it, thanks for the advice! 💯